block-producer/src/state_view/mod.rs

```diff
@@ -44,6 +47,8 @@ where
     S: Store,
 {
     // TODO: Verify proof as well
+    #[allow(clippy::blocks_in_conditions)] // Workaround of `instrument` issue
+    #[instrument(skip(self, candidate_tx), err, fields(account_id = %candidate_tx.account_id().to_hex()))]
     async fn verify_tx(
```

Comment:
Should we use `skip(all)` here?

Also, I don't think we need to add `accountId` here. If we want to log account ID, we should do it on `add_transcation()` as mentioned in the previous comment.